Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCreator.swift

```diff
@@ -8,8 +8,7 @@ struct PPOProjectCreator: View {
 
   var body: some View {
     HStack(alignment: .firstTextBaseline) {
-      // TODO: Localize
-      Text("Created by **\(self.creatorName)**")
+      Text("\(Strings.project_menu_created_by()) **\(self.creatorName)**")
```

Comment:
I don't think this is blocking release since it looks like we got lucky, but we should never combine translated strings like this; instead, the localized string should take in the creator name as input (there's no guarantee that this order of words makes sense in other languages).  See https://github.com/kickstarter/ios-oss/blob/459b37f241b0bf0fa0f898c6c8c6402be013a7f2/Library/Strings.swift#L25476 for an example of what this should look like.

Would you file a ticket for this? I think this one might get a little extra complicated since we also want the creator name formatted differently.

Reply:
Good callout. [Ticketed](https://kickstarter.atlassian.net/issues/MBL-1879).